engine/sound/src/devices/device_js.cpp

```diff
@@ -87,6 +87,7 @@ namespace dmDeviceJS
         assert(info);
         JSDevice *dev = (JSDevice*) device;
         info->m_MixRate = dmGetDeviceSampleRate(dev->devId);
+        info->m_FrameCount = info->m_MixRate * 0.025;
```

Comment:
Hmm, well, I wanted to remove this hardcoded constant, which was why I wanted the frame count configuration.
(the property was for developer to use.

And, any runtime manipulation/calculation should more likely happen further "up" in the callstack, not at this level.

So, if you're not going to use the setting, I suggest coming up with a better (documented) calculation in the code.
Sure, it works, but we want as stable solutions as possible (which means we need to calculate a better frame count).